client/app/components/game-client.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
+    GameChannel.on('event', payload => {
+      this.putMessage({
+        server: {
```

Comment:
I think I sorted this out, and naturally it would be as I laid down for bed.

So handle bar templates check for truthy, by making the message a hash that either has a server property like here or a client property I can then differentiate them and render it correctly. I think this reads much better.

This also means we can add more to the server/client hash going forward beyond the text field as we get more meta data from the server etc.